hack/make/.ensure-images

```diff
@@ -0,0 +1,9 @@
+for image in `docker images | awk '{print $1}'`; do
+if ( [ -z "${image##ppc64le/busybox}" ] || [ -z "${image##s390x/busybox}" ] ); then
+	docker tag $image busybox:latest
+	docker rmi $image
+elif ( [ -z "${image##ppc64le/hello-world}" ] || [ -z "${image##s390x/hello-world}" ] ); then
+	docker tag $image:frozen hello-world:frozen
+	docker rmi $image:frozen
+fi
+done
```

Comment:
This script is really hacky, and makes adding more architectures or more test images even more of a pain than it already is going to be -- can you think of some way we could centralize more of this in some way?  Ideally, we should only have to specify the mapping of "this organization account on the Hub corresponds to the top-level official images for this specific architecture" in a single place and have things like this fixed for us more naturally.

I also don't understand why this PR still has new `Dockerfile`s in it when I've said many times before now that I'd like to see CI working on these architectures _without_ using Docker to build (ie, `AUTO_GOPATH=1 ./hack/make.sh ...` the same way new systems have to be bootstrapped and the way packagers compile Docker) before we try building and testing inside Docker.  Since we still don't have multi-arch support in the Hub, I don't think it's a great idea for us to rush into any of this and get too stuck on one particular implementation.  The smaller we can make the changes for multi-arch, the better I'll feel about the project being able to support it in the long term.  This PR as-is is going to add pretty significant maintenance burden on the project with each new architecture we try to add.